ote/src/cljs/ote/views/service_search.cljs

```diff
@@ -62,43 +62,58 @@
 
 (def external-interface-table-columns
   ;; [label width value-fn]
-  [[:table-header-external-interface "10%"
+  [[:table-header-external-interface "20%"
     (comp #(common-ui/linkify % % {:target "_blank"}) ::t-service/url ::t-service/external-interface)]
    [::t-service/format-short "10%" ::t-service/format]
    [::t-service/license "10%" ::t-service/license]
    [::t-service/external-service-description "10%"
     (comp #(t-service/localized-text-for "FI" %) ::t-service/description ::t-service/external-interface)]])
 
+(defn parse-content-value [value-array]
+  (let [data-content-value #(str (tr [:enums ::t-service/interface-data-content %]) ", ")
+        value-str (apply str (map #(data-content-value %) value-array))
+        return-value (if (< 45 (count value-str))
+                       (str (subs value-str 0 45) "...")
+                       value-str)]
+    return-value))
+
 (defn- external-interface-links [e! {::t-service/keys [id external-interface-links name
                                                        transport-operator-id ckan-resource-id]}]
-  (when-not (empty? external-interface-links)
-    [:div
-     [:span.search-card-title (tr [:service-search :external-interfaces])]
-     [:table {:style {:margin-top "10px"}}
-      [:thead (stylefy/use-style style/external-interface-header)
-       [:tr
-        (doall
-          (for [[k w _] external-interface-table-columns]
-            ^{:key k}
-            [:th {:style {:width w :height "20px"}}
-             (tr [:field-labels :transport-service-common k])]))]]
-      [:tbody (stylefy/use-style style/external-interface-body)
-       (doall
-         (map-indexed
-           (fn [i row]
-             ^{:key i}
-             [:tr {:selectable false :style style/external-table-row}
-              (doall
-                (for [[k w value-fn] external-interface-table-columns]
-                  ^{:key k}
-                  [:td {:style {:width w :font-size "14px"}}
-                   (value-fn row)]))])
-           external-interface-links))]]]))
+  (let [data-content-value #(str (tr [:enums ::t-service/interface-data-content %]) ", ")]
```

Comment:
Is this `data-content-value` used anywhere?

Reply:
not anymore. removed